src/heartbeat/listener.js

```diff
@@ -52,7 +52,6 @@ function handleHeartbeatResponse(err, res, refocusInstanceObj=null) {
       flushTimeout: config.collectorConfig.sampleUpsertQueueTime,
       verbose: false,
       flushFunction: httpUtils.doBulkUpsert,
```

Comment:
You removed refocus instance from queue. Then please update queueUtils.js and httpUtils.js which are using refocus instance to post data.

Reply:
ok

Reply:
updated.